components/layout/side-bar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { AppBar } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+
+import { Consumer } from './context'
```

Comment:
Maybe `withGlobalStore` would be better.

Reply:
I do not get it. Is it React API function?

In this case this is not about global. ID and state passing is only implementation details for accessibility reason. It should not be part of any external API. I used context to property hide such detail.

Reply:
Sorry, I thought you were exporting it from `/store` directory. It works like HOC.

https://github.com/oacore/dashboard/blob/master/store/index.jsx#L16
https://github.com/oacore/dashboard/blob/master/store/index.jsx

Anyway, do you really want to create multiple contexts/stores in the application? I have very very very bad experience to doing so. (hard to debug, hard to understand the code...) I would rather split the global store into multiple modules.

Reply:
I do. I am just playing with technologies in this point. I do not really like React approach for such kind of things.

Reply:
Then tell me why we needed MobX when you don't want to use it? We can have extra context for every component and not care at all about global state. I just have a bad feeling about it but not forcing you to change it. It's up to you.